main.rb

```diff
@@ -0,0 +1,135 @@
+require './app'
+def create_student(app)
+  puts 'Let\'s create a student!'
+  puts 'What is your name?'
+  name = gets.chomp
+  puts 'What is your age?'
+  age = gets.chomp.to_i
+  puts 'Has parents permission? (y/n)'
+  parent_permission = gets.chomp.downcase == 'y'
+  app.create_student(age: age, name: name, parent_permission: parent_permission)
+  puts 'Student created!'
+end
```

Comment:
- [ ] When I am creating a student, I am getting the error below 👇 please fix it 

![Screen Shot 2022-06-17 at 13 23 04](https://user-images.githubusercontent.com/75126481/174289457-3820ebf3-819f-48b0-a634-fec3e34d7f67.png)

Reply:
Well resolved. I created a student successfully!  👍🏽